app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -155,13 +161,21 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         super.onStart()
 
         sessionFeature?.start()
+        serviceLocator!!.sessionRepo.events.subscribe {
+            when (it) {
+                SessionRepo.Event.YouTubeBack -> youtubeBackHandler.handleBackClick()
+                SessionRepo.Event.ExitYouTube -> youtubeBackHandler.goBackBeforeYouTube()
+                null -> return@subscribe
```

Comment:
It can't, but the compiler doesn't believe me (frustratingly, nullability annotations are inconsistently applied in RxJava).  Maybe I should add a comment about that.